src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,16 +53,28 @@ public PersonCard(Person person, int displayedIndex) {
         email.setText(person.getEmail().value);
         description.setText(person.getDescription().value);
         String tutorialsText = person.getTutorials().stream()
-            .map(t -> "T" + t.getValue())
-            .sorted()
-            .collect(Collectors.joining(", "));
+                .map(t -> "T" + t.getValue())
+                .sorted()
+                .collect(Collectors.joining(", "));
         tutorials.setText(tutorialsText);
         person.getSocialMediaLinks().stream()
                 .sorted(Comparator.comparing(sm -> sm.socialMediaLink))
                 .forEach(sm -> {
-                    Label label = new Label(sm.socialMediaLink);
-                    label.setStyle("-fx-font-size: 13px; -fx-text-fill: white; -fx-font-family: 'Segoe UI Semibold';");
-                    socialMediaLinks.getChildren().add(label);
+                    Hyperlink hyperlink = new Hyperlink(sm.socialMediaLink);
```

Comment:
Needs to work with multiple links which appear on the same line

Reply:
Will implement that in another PR as it does not affect functionality compared to our old code. Does not throw any errors either.